src/main/java/com/acrolinx/sidebar/swing/AcrolinxSidebarSwing.java

```diff
@@ -62,17 +62,20 @@ public AcrolinxSidebarSwing(final AcrolinxIntegration integration, final Acrolin
     }
 
     @Override
-    protected void processKeyEvent(final KeyEvent e)
-    {
-        // Hack to prevent pasting event for editor (e. g. .
-        if ((e.getModifiersEx() == Toolkit.getDefaultToolkit().getMenuShortcutKeyMask())
-                && (e.getKeyCode() == KeyEvent.VK_V)) {
-            // Consume it.
-            e.consume();
-        } else {
-            super.processKeyEvent(e);
+    protected void processKeyEvent(final KeyEvent keyEvent)
+    {
+        // Consume all paste events (CTRL+V) in the sidebar to prevent inserting the content into the editor
+        // too.
+        if ((keyEvent.getKeyCode() == KeyEvent.VK_V) && isMetaOrCtrlModifier(keyEvent)) {
```

Comment:
you can omit one layer of parenthesis around the first condition:
`        if (keyEvent.getKeyCode() == KeyEvent.VK_V && isMetaOrCtrlModifier(keyEvent)) {`